ballerina/tests/30_unary_oauth2_client.bal

```diff
@@ -52,3 +52,233 @@ public isolated function testStringValueReturnWithOauth2() returns Error? {
         test:assertEquals(unionResp, "Hello WSO2");
     }
 }
+
+@test:Config {enable:true}
+public isolated function testStringValueReturnWithOauth2PasswordGrantConfig() returns Error? {
```

Comment:
Please change all the places and update `Oauth` to `OAuth`. If it is there as the prefix of a variable, it should be `oauth`.
```suggestion
public isolated function testStringValueReturnWithOAuth2PasswordGrantConfig() returns Error? {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/333/commits/571ad879e6eda23cdc012b3de32e3503063ffab7